subworkflows/local/align_short.nf

```diff
@@ -58,8 +58,8 @@ workflow ALIGN_SHORT {
 
 
     // Convert merged BAM to CRAM and calculate indices and statistics
+    SAMTOOLS_SORMADUP.out.bam.view()
     SAMTOOLS_SORMADUP.out.bam
-    | map { meta, bam -> [ meta, bam, [] ] }
     | set { ch_stat }
```

Comment:
I think you can simply skip that and directly use `SAMTOOLS_SORMADUP.out.bam` in the `emit:` block below.